src/views/Designs.vue

```diff
@@ -308,6 +315,46 @@ export default Vue.extend({
     },
     toggleLoader() {
       this.isDeleting = !this.isDeleting;
+    },
+    visualize() {
+      this.selected.forEach(design => {
+        // TODO: Associate design id with the card
+        const card = {
+          uuid: uuidv4(),
+          name: design.name,
+          organism: this.organism(this.model(design.model_id).organism_id),
+          modelId: design.model_id,
+          method: "pfba", // TODO - should this be default?
+          dataDriven: false,
+          // Design card fields
+          objective: {
+            reaction: null,
+            maximize: true
+          },
+          reactionAdditions: design.design.reactionKnockins,
+          reactionKnockouts: design.design.reactionKnockouts,
+          geneKnockouts: design.design.geneKnockouts,
+          editedBounds: design.design.constraints,
+          // Data-driven card fields
+          experiment: null,
+          condition: null,
+          conditionData: null,
+          conditionWarnings: [],
+          conditionErrors: [],
+          // General simulation fields
+          isSimulating: false,
+          hasSimulationError: false,
+          growthRate: null,
+          fluxes: null
+        };
+        // Make sure the full model is available before adding the card.
```

Comment:
You say this is necessary for Escher? Otherwise it'd be neat for the browser to start drawing the interactive map component while still fetching the full model and then initializing the Escher map when a model is ready.

Reply:
I absolutely agree. The main reason to delay the navigation is because there's logic in the interactive map component that needs to know whether there are already cards in store (see comments on [this commit](https://github.com/DD-DeCaF/caffeine-vue/commit/7ad284bd17b49ce7ba4d4fc37626afdf328858a3)).

If we don't wait, we'd need different logic to select the correct card, and make sure not to add a default since we're sharing designs from another component. I guess that should be possible somehow?

Reply:
I'll think about it :slightly_smiling_face: